pkg/tests/ossm-federation/testdata/traffic-splitting/west-mesh/smcp_custom_cert.yaml

```diff
@@ -89,9 +88,11 @@ spec:
                 memory: 128Mi
               limits: {}
   security:
+    identity:
```

Comment:
Same here: ThirdParty is only when we execute in ROSA

Reply:
Nope, this test also set the same thing before I refactored it: https://github.com/maistra/maistra-test-tool/blob/maistra-2.4/testdata/examples/federation/install_diff_cert.sh#L34